projects/translation/index.Rmd

```diff
@@ -0,0 +1,97 @@
+---
+title: Translating R Messages/Warnings/Errors # Warning: changing the title may create a new Discussions thread!
+description: "" # Optional short description for post on Discussions
+author: María Nanton and Saranjeet Kaur # and co-authors if applicable
+output: html_document
+categories: [Translation] # use labels from https://github.com/r-devel/r-project-sprint-2023/labels
+comments:
+  giscus: 
+    repo: "r-devel/r-project-sprint-2023"
+    repo-id: "R_kgDOIhAibA"
+    category: "Proposals"
+    category-id: "DIC_kwDOIhAibM4CW3GY"
+    mapping: "title"
+    reactions-enabled: true
+    loading: lazy
+---
+
+```{=html}
+<!-- 
+This template is based on https://contributor.r-project.org/r-project-sprint-2023/projects/quartz-alpha-mask/.
+The sections are provided as a guide and may not be appropriate to your proposal, feel free to skip or change sections.
+
+Please label any R code chunks, especially those producing images.
+-->
+```
+```{r setup}
+
+```
+
+## Background
+
+R supports translation of messages, errors and warnings from English to other languages. This relies on contributed translations for the message strings. This project will gather contributors to work both on translations and guidelines for translation teams.
+
+Contributors to this project will work closely with sprint participants working on other projects related to translation.
+
+## Problem statement
+
+Although the message translations for some languages are well-maintained, others have become out of date, with many incorrect or missing translations. (See the post [R Can Use Your Help: Translating R Messages](https://blog.r-project.org/2022/07/25/r-can-use-your-help-translating-r-messages/index.html) on the R Blog for more information). There are other widely-spoken languages that are currently not covered at all.
+
+In addition, the process of contributing messages has changed. The new [Weblate server](https://translate.rx.studio/projects/r-project/) provides a user-friendly interface for contributing translations, making it easier for R users to contribute. However there is a need to document the process of contribution and provide guidelines for contributors, to ensure quality and consistency.
```

Comment:
This may *wrongly* suggest that the only way to contribute is via Weblate.    Which is not at all true.
The last two non-small translation contributions to R where for 'de' and 'ru' and were done 100% traditionally: 
Just be e-mailing to R-core  a tar ball say (all_ru.tar.gz) of all `*.po` files for Russian in way that they can simply be unpacked within the the R (i.e "R-devel") source tree.   Note that this traditional way of contributing is most efficient in case the translator/translation team feels comfortable with it.
Consequently instead of   "has changed"  I think you should say (something like)
 "_has been enhanced with a web-based user-friendly translation interface_"

Reply:
Good point, incorporating your suggestion as follows:

```suggestion
In addition, the process of contributing messages has recently been enhanced with a web-based, user-friendly, translation interface. The [Weblate server](https://translate.rx.studio/projects/r-project/) makes it easier for R users to contribute. However there is a need to document the process of contribution and provide guidelines for contributors, to ensure quality and consistency.
```